include/mercury/kw/kw.h

```diff
@@ -33,7 +33,9 @@ class KanoWorld
      *
      * \param client    (Optional) The HTTP client to use for requests
      */
-    explicit KanoWorld(std::shared_ptr<IHTTPClient> client =
+    explicit KanoWorld(
```

Comment:
Do we want a rule of thumb where we use `explicit` only for single argument constructors or use it for all constructors with arbitrary no. of constructors? Is there a different reason why this one was made `explicit`?

Reply:
I upped it from a one parameter to a two parameter but never ditched the `explicit`. Fixed.